packages/lib/src/components/internal/SecuredFields/lib/securedField/SecuredField.ts

```diff
@@ -97,6 +98,10 @@ class SecuredField extends AbstractSecuredField {
         // Set result back onto config object
         this.sfConfig.iframeUIConfig.ariaConfig = processedAriaConfig;
 
+        // console.log('### SecuredField::init:: processedAriaConfig', processedAriaConfig);
+
+        console.log('### SecuredField::init:: processedAriaConfig', processedAriaConfig);
```

Comment:
Is it intentional to leave the console.log here?

Reply:
Nope!